test-cases/manager/manager-backup-and-restore-5TB-single-node.yaml

```diff
@@ -0,0 +1,28 @@
+test_duration: 4320
+
+prepare_write_cmd: ["cassandra-stress write cl=ONE n=1342177280 -schema 'replication(factor=1) compaction(strategy=LeveledCompactionStrategy)' -mode cql3 native  -rate threads=50 -col 'size=FIXED(64) n=FIXED(16)' -pop seq=1..1342177280",
+                    "cassandra-stress write cl=ONE n=1342177280 -schema 'replication(factor=1) compaction(strategy=LeveledCompactionStrategy)' -mode cql3 native  -rate threads=50 -col 'size=FIXED(64) n=FIXED(16)' -pop seq=1342177281..2684354560",
+                    "cassandra-stress write cl=ONE n=1342177280 -schema 'replication(factor=1) compaction(strategy=LeveledCompactionStrategy)' -mode cql3 native  -rate threads=50 -col 'size=FIXED(64) n=FIXED(16)' -pop seq=2684354561..4026531840",
+                    "cassandra-stress write cl=ONE n=1342177280 -schema 'replication(factor=1) compaction(strategy=LeveledCompactionStrategy)' -mode cql3 native  -rate threads=50 -col 'size=FIXED(64) n=FIXED(16)' -pop seq=4026531841..5368709120"]
+
+stress_read_cmd: ["cassandra-stress read cl=ONE n=1342177280 -schema 'replication(factor=1) compaction(strategy=LeveledCompactionStrategy)' -mode cql3 native  -rate threads=50 -col 'size=FIXED(64) n=FIXED(16)' -pop seq=1..1342177280",
+                  "cassandra-stress read cl=ONE n=1342177280 -schema 'replication(factor=1) compaction(strategy=LeveledCompactionStrategy)' -mode cql3 native  -rate threads=50 -col 'size=FIXED(64) n=FIXED(16)' -pop seq=1342177281..2684354560",
```

Comment:
In this case it's actually good to leave it cl=ONE even if we increase the cluster size (and we should) for the splitted test.
Because reading with cl=ONE can catch data loss / data consistency issues.